frontend/src/views/Join.tsx

```diff
@@ -80,6 +96,18 @@ function JoinGamePage() {
             {nickname}
             &quot;.
           </LargeText>
+          <div>
+            {
+              users.map((user) => (
+                <UserNicknameText onClick={(event) => {
+                  deleteUser((event.target as HTMLElement).innerText);
+                }}
+                >
```

Comment:
Nit: is it normal to put the angle bracket on a separate line when it has the same indentation as the closing brackets above it?

Reply:
Yes -- this is actually a required ESLint styling when an inner attribute function is called. (See image below for an example of this error being thrown.)

<img width="545" alt="ESLint Styling Errors" src="https://user-images.githubusercontent.com/7987279/93248154-52f75880-f744-11ea-9cb3-c085d5182c9d.png">